src/pycram/robot_description.py

```diff
@@ -167,6 +173,37 @@ def get_actuator_for_joint(self, joint: str) -> Optional[str]:
             return self.joint_actuators.get(joint)
         return None
 
+
+    def get_actuated_joint_names(self, part_name) -> Optional[str]:
+        """
+        Given a part name, returns the actuated joints.
+
+        Args:
```

Comment:
Stick to the rst format for documentation of classes. Look at the other methods for reference

Reply:
done